tools/scripts/update-bumpenvs-yaml.py

```diff
@@ -29,6 +29,9 @@
 import requests
 import yaml
 
+NON_EXISTENT_IMAGE = (
+    "determinedai/environments:cuda-11.0-pytorch-1.7-tf-2.4-gpu-5f6f6e1"
```

Comment:
Why is this hardcoded here?

Reply:
... no idea. I suspect this happened when I ran the script and it operated on itself maybe? Will investigate.

Reply:
This is gone after ditching the NON_EXISTENT_IMAGE business for just using the new tag as old...